pkg/controller/cloudlet_api.go

```diff
@@ -663,6 +663,9 @@ func (s *CloudletApi) createCloudletInternal(cctx *CallContext, in *edgeproto.Cl
 		in.RootLbFqdn = getCloudletRootLBFQDN(in)
 		in.StaticRootLbFqdn = in.RootLbFqdn
 		if features.IsSingleKubernetesCluster {
+			if in.EnableDefaultServerlessCluster {
+				return errors.New("cannot enable default serverless cluster for single kubernetes cluster platform")
+			}
```

Comment:
I am confused - aren't these two equivalent? Or a single k8s clusters can be defaultServerlessCluster

Reply:
They're somewhat different. The defaultCluster is a "fake" cluster that represents the entire cloudlet. Because the cloudlet platform doesn't actually support CreateCluster, we create this cluster via direct STMPut into Etcd. The defaultMTCluster is a MT cluster meant to co-exist with other dedicated-tenant clusters on a Cloudlet that supports CreateCluster, so it gets built the normal way. Resource handling is also slightly different, for the solo case, the platform dedicates all resources to the fake cluster. For the MT case, it just gets some default set of resources.

Reply:
Thanks for the explanation.